gdsfactory/add_ports.py

```diff
@@ -173,18 +173,17 @@ def add_ports_from_markers_center(
         port_name = f"{port_name_prefix}{i+1}" if port_name_prefix else str(i)
         bbox = p.bbox()
```

Comment:
we can also use `p.dbbox()` here so that we don't have to deal with dbu conversion